README.md

```diff
@@ -50,6 +50,7 @@ Attributes
 * `node["tomcat"]["ssl_cert_file"]` - SSL certificate file
 * `node["tomcat"]["ssl_chain_files"]` - SSL CAcert chain files used for generating the SSL certificates
 * `node["tomcat"]["ssl_max_threads"]` - maximum number of threads in the ssl connector pool, default `150`.
+* `node["tomcat"]["ssl_enabled_protocols"]` - SSL enabled protocols.
```

Comment:
I would make a recommendation in the description to use 'TLSv1.2,TLSv1.1,TLSv1' or a smaller subset, to mitigate poodle attack issues via SSL.